### ISSUE TITLE
pass column names to `:fastmath/ols` models

### ISSUE THREAD

**Issue by daslu** [scicloj/metamorph.ml#24]
`fastmath.ml.regression/lm` supports a `:names` option that controls the names of the coefficients when the model is printed.
https://generateme.github.io/fastmath/clay/ml.html#lm

In the `metamorph.ml` context, our data originate from a dataset, so we may use the column names for that.

**Comment by behrica** [comment]
@daslu 
did you mention this as one of the changes you wanted to do, if /when we would add the model to tablecloth itself ?

(which I am not sure, we should do, as the model is available here)

**Comment by daslu** [comment]
Sorry for missing this message earlier.

Yes, I think we should have a Tablecloth function to take care of that eventually.

**Comment by behrica** [comment]
are you refering to pass :names as written here:
https://generateme.github.io/fastmath/clay/ml.html#lm

If so, then fixing #27 will fix this as well.

**Comment by daslu** [comment]
Yes

**Comment by daslu** [comment]
Yes, but I am proposing this will happen automatically -- the dataset column names will be passed as `:names` by default.

**Comment by behrica** [comment]
@daslu 
I  don't understand what you mean with this change request.
Maybe you can do an PR ?

**Comment by daslu** [comment]
Thanks, good idea, I will do a PR.

(I mean something like we did in Tablemath:
https://github.com/scicloj/tablemath/blob/a667be6/src/scicloj/tablemath/v1/api.clj#L242
The option map to `lm` can automatically receive the column names of the dataset.)
